File: 多语言工具/KLanguageTool/KLanguageTool/ViewController.m
//
//  ViewController.m
//  KLanguageTool
//
//  Created by kongyulu on 2020/12/26.
//

#import "ViewController.h"
#import "KLangaugeWindowController.h"

@interface ViewController ()

@end

@implementation ViewController


- (void)viewDidLoad {
    [super viewDidLoad];

}

- (IBAction)btnStringToExcelClicked:(id)sender {
}

- (IBAction)btnExcelToStringClicked:(id)sender {
    KLangaugeWindowController *vc = [[KLangaugeWindowController alloc] init];
    
}


- (void)setRepresentedObject:(id)representedObject {
    [super setRepresentedObject:representedObject];

    // Update the view, if already loaded.
}


@end

File: Mac slic/KRemoveNoiseDemo/KRemoveNoiseDemo/ViewController.swift
//
//  ViewController.swift
//  KRemoveNoiseDemo
//
//  Created by kongyulu on 2020/9/17.
//

import Cocoa

class ViewController: NSViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        // Do any additional setup after loading the view.
        let tool = NoiseTool()
        tool.testNoiseRemove()
    }

    override var representedObject: Any? {
        didSet {
        // Update the view, if already loaded.
        }
    }


}

